source/_docs/z-wave/controllers.markdown

```diff
@@ -9,9 +9,24 @@ You need to have a compatible Z-Wave stick or module installed. The following de
 
 {% warning %}
 
-Until recently, 700 series Z-Wave Controllers had a bug that could cause the mesh to be flooded on some networks and the controller to become unresponsive. At present, all 700 series controllers share the same firmware and are subject to this bug. It appears that this bug is largely, if not completely, resolved as of firmware version 7.17.2.
-
-Users should upgrade the firmware on all 700 series controllers to version 7.17.2 or greater. Firmware can be upgraded using the below directions:
+The firmwares of 700 and 800 series Z-Wave controllers have several bugs which impact the stability of the mesh and can cause the controller to become unresponsive. Because there is no known firmware version that is completely fixed, it is recommended to choose a firmware based on the following criteria:
+- 700 series:
+  - prefer SDK versions 7.17.2 to 7.18.x
+  - SDK versions 7.19.x are okay
+  - avoid SDK versions before 7.17.2
+  - avoid SDK versions 7.20 to 7.21.3
+ 
+- 800 series
+  - prefer SDK versions 7.22.x
+  - SDK versions 7.17.2 to 7.19.x are okay
+  - avoid SDK versions before 7.17.2
+  - avoid SDK versions 7.20 to 7.21.3
+ 
+Note that the SDK version does not have to match the firmware version. If you are unsure which SDK versions a firmware is based on, contact the manufacturer.
+
+Users should upgrade the firmware on all 700 and 800 series controllers to a recommended version.
```

Comment:
If it implies a requirement for success we should wrap it in an important box.

Additionally, small adjustment toward the user from reading it.

```suggestion
{% important %}
You should upgrade the firmware on all 700 and 800 series controllers to a recommended version.
{% endimportant %}
```